components/centraldashboard/license_info.csv

```diff
@@ -0,0 +1,1287 @@
+bn.js@4.11.8,https://github.com/indutny/bn.js/tree/master/README.md,MIT,https://raw.githubusercontent.com/indutny/bn.js/master/README.md
+body-parser@1.19.0,https://github.com/expressjs/body-parser/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/expressjs/body-parser/master/LICENSE
+bonjour@3.5.0,https://github.com/watson/bonjour/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/watson/bonjour/master/LICENSE
+boolbase@1.0.0,https://github.com/fb55/boolbase/tree/master/README.md,ISC,https://raw.githubusercontent.com/fb55/boolbase/master/README.md
+boxen@1.3.0,https://github.com/sindresorhus/boxen/tree/master/license,MIT,https://raw.githubusercontent.com/sindresorhus/boxen/master/license
+brace-expansion@1.1.11,https://github.com/juliangruber/brace-expansion/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/juliangruber/brace-expansion/master/LICENSE
+braces@2.3.2,https://github.com/micromatch/braces/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/micromatch/braces/master/LICENSE
+braces@3.0.2,https://github.com/micromatch/braces/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/micromatch/braces/master/LICENSE
+brorand@1.1.0,https://github.com/indutny/brorand/tree/master/README.md,MIT,https://raw.githubusercontent.com/indutny/brorand/master/README.md
+browserify-aes@1.2.0,https://github.com/crypto-browserify/browserify-aes/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/crypto-browserify/browserify-aes/master/LICENSE
+browserify-cipher@1.0.1,https://github.com/crypto-browserify/browserify-cipher/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/crypto-browserify/browserify-cipher/master/LICENSE
+browserify-des@1.0.2,https://github.com/crypto-browserify/browserify-des/tree/master/license,MIT,https://raw.githubusercontent.com/crypto-browserify/browserify-des/master/license
+browserify-rsa@4.0.1,https://github.com/crypto-browserify/browserify-rsa/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/crypto-browserify/browserify-rsa/master/LICENSE
+browserify-sign@4.0.4,https://github.com/crypto-browserify/browserify-sign/tree/master/LICENSE,ISC,https://raw.githubusercontent.com/crypto-browserify/browserify-sign/master/LICENSE
+browserify-zlib@0.2.0,https://github.com/devongovett/browserify-zlib/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/devongovett/browserify-zlib/master/LICENSE
+browserslist@4.7.0,https://github.com/browserslist/browserslist/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/browserslist/browserslist/master/LICENSE
+buffer-alloc-unsafe@1.1.0,https://github.com/LinusU/buffer-alloc-unsafe/tree/master/readme.md,MIT,https://raw.githubusercontent.com/LinusU/buffer-alloc-unsafe/master/readme.md
+buffer-alloc@1.2.0,https://github.com/LinusU/buffer-alloc/tree/master/readme.md,MIT,https://raw.githubusercontent.com/LinusU/buffer-alloc/master/readme.md
```

Comment:
https://github.com/LinusU/buffer-alloc/tree/master/readme.md doesn't seem to be right. No license content found in it.

There is

https://github.com/LinusU/buffer-alloc/tree/master/LICENSE file.

Reply:
Problem is they do now, but not when the release was made (ie. the version at which this package is installed): https://github.com/LinusU/buffer-alloc/tree/a0d55382c805f3b117a055e0be203f13f2519cc5